src/effects/backends/effectsbackendmanager.cpp

```diff
@@ -0,0 +1,115 @@
+#include "effects/backends/effectsbackendmanager.h"
+
+#include "control/controlobject.h"
+#include "effects/backends/builtin/builtinbackend.h"
+#include "effects/backends/effectprocessor.h"
+#ifdef __LILV__
+#include "effects/backends/lv2/lv2backend.h"
+#endif
+#include "effects/presets/effectpreset.h"
+
+EffectsBackendManager::EffectsBackendManager() {
+    m_pNumEffectsAvailable = std::make_unique<ControlObject>(
+            ConfigKey("[Master]", "num_effectsavailable"));
```

Comment:
```
Warning [Main]: The following 1 controls were leaked:
Warning [Main]: "[Master]" "num_effectsavailable" ControlObject(0x5615248b31c0)
DEBUG ASSERT: "!"Controls were leaked!"" in function void MixxxMainWindow::finalize() at /home/jan/Projects/mixxx/src/mixxx.cpp:781
Aborted (core dumped)
```

Reply:
I looked into this before and couldn't figure out why this isn't getting deleted. Help would be appreciated.

Reply:
I finally found the source of this leak and fixed it: 20a2a75